app/main/util/dto.py

```diff
@@ -87,3 +87,12 @@ class TagDto:
 		'tag_id': fields.Integer(required=True, description="id of the concerned tag"),	
 		'value': fields.Integer(required=True, description="priority level of the tag")
 	})
+
+class IssueDto:
+	api = Namespace('issue', description="for issue related operations")
+	issue = api.model('tag', {
+		'cover' : fields.Integer(required=True, description="Cover image of the concerned issue"),
```

Comment:
You also need to send back, `id`, `issue_tag`. It is mentioned in the specifications sheet.

Reply:
@grapheo12 and I discussed that ID and issue_tag are autogenerated and don't have to be present in the API model.